keyboards/ekow/akira/rules.mk

```diff
@@ -0,0 +1,20 @@
+# MCU name
+MCU = STM32F042
+
+# Bootloader selection
+BOOTLOADER = stm32-dfu
+
+# Build Options
+# change yes to no to disable
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no		# Mouse keys
+<<<<<<< HEAD
```

Comment:
same issue here

Reply:
Sorry I missed those parts, thank you for your time.